app/javascript/stylesheets/overlays/alert.scss

```diff
@@ -0,0 +1,14 @@
+$btn-text-color: #fff;
+$btn-bg-color: #2f3941;
+$btn-bg-ative-color: #49545c;
+
+.neeto-ui-btn--style-danger {
+  color: $btn-text-color;
+  background-color: $btn-bg-color;
+}
+
+.neeto-ui-btn--style-danger:active,
+.neeto-ui-btn--style-danger:hover:not(:disabled) {
+  color: $btn-text-color;
+  background-color: $btn-bg-ative-color;
+}
```

Comment:
Let's use neetoUI defaults as is. No need to override it. 
Also, update neetoUI to the latest version. 
Also please look into https://www.learnhowtoprogram.com/user-interfaces/building-layouts-preprocessors/7-1-sass-architecture for understanding as to how to structure SCSS when needed.